classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_students(student)
+    students << student
```

Comment:
- [ ] Please ensure that you put `@` symbol in front of your `students` variable in your `add_students` method, that way you will be referring to your class attributes. A possible solution is: `@students << student` :slightly_smiling_face: